docs/src/developer/reference/config-options.md

```diff
@@ -946,6 +946,17 @@ brig:
       setAuditLogEmailRecipient: security@wire.com
 ```
 
+### Ephemeral User Creation
+
+Ephemeral user creation is enabled per default. To disable the feature set `setEphemeralUserCreationEnabled` to `false`:
```

Comment:
I'm wondering if all readers of the docs know what an "ephemeral user" is. :thinking: Maybe, some short explanation would be good.